event/src/main/java/com/bazaarvoice/emodb/event/core/DefaultEventStore.java

```diff
@@ -366,7 +366,7 @@ public boolean accept(EventId eventId, ByteBuffer eventData) {
                 return true;
             }
         };
-        _readerDao.readAll(channel, eventSink, since);
+        _readerDao.readAll(channel, eventSink, since, false);
```

Comment:
previously, week paramerer had been set to `true`.
do we need to add tests to verify that?

Reply:
There was no test previously, and the main point of this PR was to change it from `true` to `false`. If you really want a test, then I can add one, but to me it seems unneeded.